gen/arpresolv.c

```diff
@@ -97,6 +97,12 @@ static int getifinfo(const char *, int *, uint8_t *);
 		}							\
 	} while (/* CONSTCOND */ 0)
 
+#ifdef STANDALONE_TEST
+#define	__diagused
```

Comment:
`__diagused` is usually for `DIAGNOSTIC` so using it for `STANDALONE_TEST` is confusable. How about `__testused` or something?

Reply:
OK. I'll modify it.

Reply:
done.